node-server/src/services/websockets.js

```diff
@@ -0,0 +1,50 @@
+const wsServer = require('../sockets/ws-serv')
+const wsClient = require('../sockets/ws-client')
+
+class WebsocketService {
```

Comment:
Just as a comment, this class has no own/internal state (a real question, since I'm not that fluent in JS)? The sockets are defined in the module namespace and all instances of the class use the same socket-objects?